staging/src/k8s.io/kubectl/pkg/validation/schema.go

```diff
@@ -143,10 +143,10 @@ func (c *paramVerifyingSchema) ValidateBytes(data []byte) error {
 		case metav1.FieldValidationStrict:
 			return c.schema.ValidateBytes(data)
 		case metav1.FieldValidationWarn:
-			klog.Warningf("cannot perform warn validation if server-side field validation is unsupported, skipping validation")
+			klog.InfoS("cannot perform warn validation if server-side field validation is unsupported, skipping validation")
```

Comment:
Is this a user-visible change? If it is, we should update the changelog entry from `NONE`.

Reply:
Done